### PR TITLE
feat: group pods by main or preview

### PR DESCRIPTION
**Pull request by Fadhil**
## Summary

In this PR, we present the connectable instances in `application instances connect` in groups of "Main" or "Preview Instances".

Selecting either "Main" or "Preview" just keeps the user in the select context. User can only move forward by selecting one of the instances.
Screenshot 2024-11-01 at 2 53 35 PM
Screenshot 2024-11-01 at 2 33 08 PM

Ticket: https://mindvalley.atlassian.net/browse/DX-831

## What's Changed

Updated the `application instances connect` Select menu to display pods grouped by "is_preview"

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/232#" title="Author self-approved">Fadhil</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by onimsha** [comment]
image
 
I thought it wasn't possible, but it is. Nice work ! 🚀

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/232#" title="Author self-approved">Fadhil</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/232#pullrequestreview-2414651378" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
